DigixGlobal/governance-ui-components
# [Feature] - Add additional docs to a project

Ref [DGDG-322](https://tracker.digixdev.com/agiles/88-14/89-22?issue=DGDG-322)

This allows users to add more documents (accepts Images and PDF files) to their proposals (once Finalized).

Test Plan
 - Create a Project, have it approved by a moderator then Finalize
 - `Add Updates` button should be visible once Project is viewed
 - Add documents
 - Verify that the added documents are visible once the Project is viewed

Note:
 - PR https://github.com/DigixGlobal/governance-ui-components/pull/219 should be merged first
 - View of Additional Docs on Project details will be updated once https://github.com/DigixGlobal/governance-ui-components/pull/250 is merged
